src/plugins/cosmos/components/modals/Send/views/Details.tsx

```diff
@@ -92,7 +92,11 @@ export const Details = () => {
       </ModalHeader>
       <ModalCloseButton borderRadius='full' />
       <ModalBody>
-        <AccountDropdown assetId={asset.assetId} onChange={handleAccountChange} />
+        <AccountDropdown
+          assetId={asset.assetId}
+          onChange={handleAccountChange}
+          buttonProps={{ width: 'full', mb: 2, variant: 'solid' }}
```

Comment:
same change @reallybeard applied in https://github.com/shapeshift/web/pull/2632/commits/ab34ca025d041dd29d0fa92dcabb01eafa518edd#diff-ce0c7fcb16eddfbec35fde05724f0d01ac174912845a33a0ea6cbc65b7acead1R195